Extensions/loopyLoop.js

```diff
@@ -7,7 +7,7 @@
 /// <reference path="../globals.d.ts" />
 
 (function LoopyLoop(){
-    const bar = document.querySelector(".playback-bar .progress-bar");
+    const bar = document.querySelector(".playback-bar > div:nth-child(2)");
```

Comment:
I think `.progress-bar` was changed to `.progress-bar-wrapper` so `.playback-bar > .progress-bar-wrapper` should work and will be better if it does.

Reply:
My Spotify has no such class listing on the div I was trying to target, I think the CSS map for that class is broken, I believe spicetify maps CSS to readable names in some places.